npm/restatectl/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@restatedev/restatectl",
+  "description": "Restate administration tools",
+  "version": "0.5.1",
```

Comment:
The versioning is not in line with the Restate versioning?

Reply:
its replaced by the npm.yml scripting - the version here is just a placeholder